target-design/switch/switch.cc

```diff
@@ -34,6 +34,13 @@ int switchlat = 0;
 
 #define SWITCHLATENCY (switchlat)
 
+// param: numerator and denominator of bandwidth throttle
+// Used to throttle outbound bandwidth from port
+//
+// THESE ARE SET BY A COMMAND LINE ARGUMENT. DO NOT CHANGE IT HERE.
+int throttle_numer = 1;
+int throttle_denom = 1;
```

Comment:
are these actually set by a command line argument? I don't see it?

Reply:
Line 228. Their pointers are passed to simplify_frac, which is what sets them.

Reply:
ah got it. cool